pmd-apex/src/main/java/net/sourceforge/pmd/cpd/ApexLanguage.java

```diff
@@ -6,14 +6,16 @@
 
 import java.util.Properties;
 
+import net.sourceforge.pmd.lang.apex.ApexLanguageModule;
+
 public class ApexLanguage extends AbstractLanguage {
 
     public ApexLanguage() {
         this(new Properties());
     }
 
     public ApexLanguage(Properties properties) {
-        super("Apex", "apex", new ApexTokenizer(), ".cls");
+        super(ApexLanguageModule.NAME, ApexLanguageModule.TERSE_NAME, new ApexTokenizer(), ApexLanguageModule.EXTENSIONS);
```

Comment:
We could simplify this ctor by adding a constructor with (pmd) Language and Tokenizer

Reply:
Note that #4397 removes the CPD language class anyway...